recommendations/local-resource-conservation.md

```diff
@@ -0,0 +1,334 @@
+
+The goal of this reputation metric is to ensure that the cost to obtain 
+sufficient reputation for a HTLC to be endorsed on the outgoing channel is 
+greater than the damage that a node can inflict by abusing the access that it 
+is granted. This algorithm uses forwarding fees to measure damage, as this value 
+is observable within the protocol. It is reasonable to expect an adversary to 
+"about turn" - to behave perfectly to build up reputation, then alter their 
+behavior to abuse it. For this reason, in-flight HTLCs have a temporary 
+negative impact on reputation until they are resolved.
+
+If granted full access to a node's liquidity and slots, the maximum amount of 
+damage that can be inflicted on the targeted node is bounded by the largest 
+cltv delta from the current block height that it will allow a HTLC to set 
+before failing it with [expiry_too_far](../04-onion-routing.md#failure-messages).
+This value is typically 2016 blocks (~2 weeks) at the time of writing.
+
+Upon receipt of `update_add_htlc`, the local node considers the following to 
+determine whether the sender is classified as having sufficient reputation: 
+* Outgoing channel revenue: the total routing fees over the maximum allowed HTLC 
+  hold period (~2 weeks) that the outgoing channel has earned the local node, 
+  considering both incoming and outgoing HTLCs that have used the channel.
+* Incoming channel revenue: the total routing fees that the incoming channel has 
+  earned the local node, considering only incoming HTLCs on the channel.
+* In-flight HTLCs risk: any endorsed HTLC that the incoming channel has 
+  in-flight on the requested outgoing channel negatively affect reputation, 
+  based on the assumption that they will be held until just before their 
+  expiry height.
+
+On a per-HTLC basis, the local node will classify the sending node's 
+reputation for the offered HTLC as follows: 
+* if `incoming_channel_revenue - in_flight_risk >= outgoing_channel_revenue`: 
+  * the sending node is considered to have *sufficient* reputation for the 
+    offered HTLC.  
+* otherwise, the sending node is considered to have *insufficient* reputation 
+  for the offered HTLC.
+
+The sections that follow provide details of how to calculate the components of 
+the inequality.
+
+#### Effective HTLC Fees
+The contribution that the fees paid by a HTLC make to reputation standings is 
+adjusted by the amount of time that a HTLC took to resolve. By accounting for 
+the "opportunity cost" of HTLCs that are held for longer periods of time, the 
+reputation score rewards fast resolving, successful payments and penalizes 
+slow resolving payments (regardless of successful resolution).
+
+We define the following parameters: 
+* `resolution_period`: the amount of time a HTLC is allowed to resolve in that 
+  classifies as "good" behavior, expressed in seconds. The recommended default 
+  is 90 seconds (given that the protocol allows for a 60 second MPP timeout). 
```

Comment:
I think it would be useful to have an additional indication on the definition of the clock used to tick the `resolution_period` second and from which all the `opportunity_cost` computation are scaled on.

E.g, the Epoch which is defined on unix systems as "1970-01-01 00:00:00 +0000 (UTC), it's not a perfect clock though it's better than nothing.